core/src/components/button/button.ionic.scss

```diff
@@ -156,26 +156,9 @@
 // -------------------------------------------------------------------------------
 
 // Only show the focus ring when the button is focused
-:host(.ion-focused) {
-  --overflow: visible;
```

Comment:
This isn't needed anymore because the outline is part of the normal document flow